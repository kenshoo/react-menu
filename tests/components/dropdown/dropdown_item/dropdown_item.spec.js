import React from "react";
import renderer from "react-test-renderer";

import DropdownItem from "../../../../src/components/dropdown/dropdown_item/dropdown_item";

describe("DropdownItem component", () => {
  test("default snapshot", () => {
    const cmp = renderer.create(<DropdownItem />);
    expect(cmp).toMatchSnapshot();
  });

  test("renders children", () => {
    const cmp = renderer.create(<DropdownItem>TEST</DropdownItem>);
    expect(cmp).toMatchSnapshot();
  });

  test("className prop", () => {
    const cmp = renderer.create(
      <DropdownItem className="TEST">TEST</DropdownItem>
    );
    expect(cmp).toMatchSnapshot();
  });
});
