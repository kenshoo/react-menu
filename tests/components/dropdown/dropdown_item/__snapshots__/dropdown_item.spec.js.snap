// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownItem component className prop 1`] = `
<div
  className="sc-bdVaJa kRyzRN TEST"
>
  TEST
</div>
`;

exports[`DropdownItem component default snapshot 1`] = `
<div
  className="sc-bdVaJa kRyzRN"
/>
`;

exports[`DropdownItem component renders children 1`] = `
<div
  className="sc-bdVaJa kRyzRN"
>
  TEST
</div>
`;
